analyses/snv-callers/run_caller_consensus_analysis-pbta.sh

```diff
@@ -8,60 +8,72 @@
 set -e
 set -o pipefail
 
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
 # The sqlite database made from the callers will be called:
-dbfile=scratch/snv_db.sqlite
+dbfile=../../scratch/snv_db.sqlite
```

Comment:
When we made the S2 figures, this path got written in during review. To my knowledge figures are the _only_ thing that needs to use these databases after this module. So, we can either revert back the figures path, or save the databases there in the first place.
  
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/e72c11dbebed2377072df43150fd15f2ffa0262a/figures/scripts/supp-snv-callers-panels.R#L26

Reply:
I split the difference (kind of) in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1420/commits/054616bd42bde245e06335da4a68792633699186. It's a good call for modules to have their own directories in `scratch/`, rather than just hoping their are no file name collisions in `scratch/`! But I also think `snv-callers` is more appropriate than `snv-callers-panels`.